app/car.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+import math
+from dataclasses import dataclass
+
+
+@dataclass
+class Car:
+    brand: str
+    fuel_consumption: float
+
+    @classmethod
+    def from_dict(cls, car: dict) -> Car:
+        return cls(brand=car["brand"],
+                   fuel_consumption=car["fuel_consumption"])
+
+    def fuel_cost(self,
+                  first_point: list[int],
+                  last_point: list[int],
+                  one_liter_price: float) -> int | float:
+        return round(

```

Comment:
I believe it's generally agreed that you should choose one style of line-splitting in your project. Therefore, I'd suggest to either reformat all other instances to this* format, or reformat this one to the other**. I would personally prefer the latter, but that's up to you.

*this:
```python
round(
    ...
)
```
**other: 
```python
round(...
      ...)
```